data/sstable/loadCassandraDB.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This shell script triggers data generator and then imports the data to 
+# local Cassandra instance using cqlsh's COPY command
+
+set -e
+
+CASSANDRA_INSTALLATION_DIR="/tmp"
+CURR_DIR=$PWD
+
+# Prepare the data
+./generate-data.py
+
+# this will append a relative path to load the csv data
+if [[ $(wc -l < cassdb.cql) -le 93 ]]; then
```

Comment:
Why is this necessary?

Reply:
This will append the address of flights_data.csv at the end of cassdb.cql.
We can't put a static/predefined address in the cassdb.cql  because Hillview's root and Cassandra's root are arbitrary.

Reply:
Since cassdb.cql is in the same directory can't use just use "."?

Reply:
In this case, you don't need to modify the script at all

Reply:
Yes, the cassdb.cql is in the same dir, but the flights_data.csv is in Hillview's dir. In the last line of cassdb.cql, the Copy script will look similar to this:

COPY CassDB.flights (ID,DayOfWeek,FlightDate,UniqueCarrier,Origin,OriginCityName,OriginState,Dest,DestState,DepTime,DepDelay,ArrTime,ArrDelay,Cancelled,ActualElapsedTime,Distance)  FROM '/Users/daniar/Documents/GitPR/hillview/data/sstable/flights_data.csv' WITH DELIMITER=',' AND HEADER=TRUE; exit;